tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -621,6 +640,12 @@ LogicalResult LowerLLVMToBinary(ModuleOp module,
     return failure();
   }
 
+  if (tm->getRelocationModel() != llvm::Reloc::Model::PIC_) {
+    llvm::errs() << "-fPIC must be specified\n";
```

Comment:
This should never happen, the codes just functions as assert(), am i getting it correctly?

Reply:
Yes, it should not happen after this PR. still put this check here for double check.